bindings/src/main/java/io/pravega/storage/extendeds3/ExtendedS3Storage.java

```diff
@@ -228,6 +231,13 @@ private int doRead(SegmentHandle handle, long offset, byte[] buffer, int bufferO
 
             int bytesRead = StreamHelpers.readAll(reader, buffer, bufferOffset, length);
 
+            Duration elapsed = timer.getElapsed();
+
+            ExtendedS3Metrics.READ_LATENCY.reportSuccessEvent(elapsed);
+            ExtendedS3Metrics.READ_BYTES.add(length);
+
+            log.debug("Read  segment={} offset={} bytesWritten={} latency={}.", handle.getSegmentName(), offset, length, elapsed);
```

Comment:
1. `elapsed.toMillis`. Otherwise this will print out some weird string.
2. Extra space between "Read" and "segment"

Reply:
Good catch. Thanks.
Fixed.